### PR TITLE
[terra-menu] convert night watch to webdriver

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
References #1302. 

Note: accessibility errors were discovered for the header buttons. Issue #1585 logged.

### CODE REVIEW

File: packages/terra-menu/tests/wdio/menu-spec.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    Terra.should.matchScreenshot('returned to main menu', { selector: '#root' });
+  });
+
+  describe('Menu Keyboard Navigation-Enter Key', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-menu/menu/menu/sub-menu');
+      browser.click('#sub-menu-button');
+    });
+
+    it('displays the submenu on right arrow', () => {
```
Comment (tomleewu) [inline packages/terra-menu/tests/wdio/menu-spec.js]: I think you meant to name the test '_displays the submenu on Enter keypress_'

and the one on line 152 '_returns to the main menu on Enter keypress_'

Comment (bjankord) [reply packages/terra-menu/tests/wdio/menu-spec.js]: Yeah lets get these test names updated so the screenshots match before we merge.

**Comment by emilyrohrbough** [comment]
@tomleewu @bjankord These test descriptions have been updated. Can I get another look?

File: packages/terra-menu/tests/wdio/menu-spec.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
+/* global browser, describe, it, before, expect, Terra */
+
+const ignoredA11y = {
+  // Isse logged here: https://github.com/cerner/terra-core/issues/1585
```
Comment (tomleewu) [inline packages/terra-menu/tests/wdio/menu-spec.js]: nit: misspelling of _"Issue"_

**Pull request merged**